assets/js/components/Loadpoint.vue

```diff
@@ -338,11 +338,11 @@ export default {
 		},
 		fmtPower(value) {
 			const inKw = value == 0 || value >= 1000;
-			return this.fmtKw(value, inKw);
+			return this.fmtW(value, inKw ? POWER_UNIT.KW : POWER_UNIT.AUTO);
 		},
 		fmtEnergy(value) {
 			const inKw = value == 0 || value >= 1000;
```

Comment:
```suggestion
			const inKw = value == 0;
```

Reply:
I guess we could also move this "0 Watt should always be with kW unit" rule into the `POWER_UNIT.AUTO` behavior for the format function. I currently dont see any conflicts or sideeffects of doing this.

Reply:
Should 0 Watt always have 0 digits?

Reply:
Ne, the zero-state is `0.0 kW`

![Bildschirmfoto 2024-09-02 um 23 11 18](https://github.com/user-attachments/assets/afc68ca8-be3d-4723-b4cc-dca2d1d5f7b4)